lib/etl/scrape.js

```diff
@@ -129,8 +88,12 @@ function executePipeline(gamer, IS_BACKFILL = false) {
  * @param IS_BACKFILL
  * @returns {PromiseLike<void>}
  */
-export function refreshData(query = {}, IS_BACKFILL = false) {
-    return queryDatabase(GAMER_TABLE, query)
-        .then((gamers) =>  Bluebird.mapSeries(gamers, (gamer)=> executePipeline(gamer, IS_BACKFILL)))
-        .then(() => console.log("Finished entire refresh"));
-}
\ No newline at end of file
+export async function refreshData(query = {}, IS_BACKFILL = false) {
```

Comment:
Again, we should do more descriptive names for these methods. Could be confusing as to how much data is being refreshed, what's happening, or if it's gonna delete and get all the data again.

Reply:
The latest flavor of this looks like this. 

I think refreshData is an okay name since it's a more general API. The specific APIs are below it. 


`/**
 * First queries the gamers table,
 * then executes the pipeline for each gamer
 * @param query
 * @returns {PromiseLike<void>}
 */
export async function refreshData(query = {}) {
    let API = await login();
    let gamers = await queryGamers(query);
    await Bluebird.map(gamers, (gamer) => executePipeline(gamer, API), {concurrency: THREAD_CONCURRENCY_LIMIT});
    console.log("Finished entire refresh")
}

async function runUpdates(){
    await refreshData({needs_update: true});
}

async function runBackfills() {
    await refreshData({needs_backfill: true});
}`